doc/getting_started.rst

```diff
@@ -66,57 +66,81 @@ As a pure python package, pydicom is easy to install and has no requirements
 other than python itself (the NumPy library is recommended, but is only
 required if manipulating pixel data).
 
-.. note::
-   In addition to the instructions below, pydicom can also be installed
-   through the `Python(x,y) <https://sourceforge.net/projects/python-xy/>`_
-   distribution, which can install python and a number of packages [#]_
-   (including pydicom) at once.
 
 Prerequisites
 -------------
 
 * Python 2.7, 3.4 or later
 * Optional dependencies:
-   * numpy
-   * pillow
-   * gdcm
-   * jpeg_ls
-   * jpeg2000
+  * numpy
+  * pillow
+  * gdcm
+  * jpeg_ls
+  * jpeg2000
+  * pytest (if running pydicom's test suite)
 
```

Comment:
This should be 3 spaces to make a list (a think I changed that before).
And while you are at it - can be please also fix the code block in page ``working_with_pixel_data`` below, where the code is indented only two spaces? I made that mistake in the last check-in and don't want to make a separate PR for this.